src/components/standup/index.tsx

```diff
@@ -0,0 +1,139 @@
+import { FC, useState, useEffect } from 'react';
+
+import styles from '@/components/standup/standupContainer.module.scss';
+
+import {
+    useAddStandupMutation,
+    useUserStandupDetailsQuery,
+} from '@/app/services/standup';
+import { useGetUserQuery } from '@/app/services/userApi';
+
+import FormInputComponent from './FormInputComponent';
+import { standupUpdateType } from '@/interfaces/standup.type';
+import { getYesterdayDate } from '@/utils/getYesterdayDate';
+import { getTotalMissedUpdate } from '@/utils/getTotalMissedUpdate';
+
+const StandUpContainer: FC = () => {
+    const [standupUpdate, setStandupUpdate] = useState<standupUpdateType>({
+        type: 'user',
+        completed: '',
+        planned: '',
+        blockers: '',
+    });
+
+    const [buttonDisable, setButtonDisable] = useState<boolean>(true);
+    const [addStandup] = useAddStandupMutation();
+    const { data: user } = useGetUserQuery();
+    const { data: userStandupdata } = useUserStandupDetailsQuery(user?.id);
+    const standupDates = userStandupdata?.data?.map((element) => element.date);
+    const totalMissedUpdate = getTotalMissedUpdate(standupDates || []);
+
+    const yesterdayDate = getYesterdayDate();
+
+    const buttonStyleClass = buttonDisable
+        ? `${styles.nonActiveButton}`
+        : `${styles.activeButton}`;
+
+    const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setStandupUpdate((prevStandupUpdate) => ({
+            ...prevStandupUpdate,
+            [event.target.name]: event.target.value,
+        }));
+    };
+    const isValidate = () => {
+        return (
+            standupUpdate.completed !== '' &&
+            standupUpdate.planned !== '' &&
+            standupUpdate.blockers !== ''
+        );
+    };
+    useEffect(() => {
+        const isValid = isValidate();
+        setButtonDisable(!isValid);
+    }, [standupUpdate.completed, standupUpdate.planned]);
+
+    const handleFormSubmission = async (
+        event: React.FormEvent<HTMLFormElement>
+    ) => {
+        event.preventDefault();
+        setStandupUpdate({
+            type: 'user',
+            completed: '',
+            planned: '',
+            blockers: '',
+        });
+        await addStandup(standupUpdate);
```

Comment:
where are we checking if this fails? 

How are we responding to user in case of success or error?

Reply:
fixed